nixos/modules/system/boot/tmp.nix

```diff
@@ -1,38 +1,35 @@
 { config, lib, ... }:
-
-with lib;
-
 let
   cfg = config.boot.tmp;
 in
 {
   imports = [
-    (mkRenamedOptionModule [ "boot" "cleanTmpDir" ] [ "boot" "tmp" "cleanOnBoot" ])
-    (mkRenamedOptionModule [ "boot" "tmpOnTmpfs" ] [ "boot" "tmp" "useTmpfs" ])
-    (mkRenamedOptionModule [ "boot" "tmpOnTmpfsSize" ] [ "boot" "tmp" "tmpfsSize" ])
+    (lib.mkRenamedOptionModule [ "boot" "cleanTmpDir" ] [ "boot" "tmp" "cleanOnBoot" ])
+    (lib.mkRenamedOptionModule [ "boot" "tmpOnTmpfs" ] [ "boot" "tmp" "useTmpfs" ])
+    (lib.mkRenamedOptionModule [ "boot" "tmpOnTmpfsSize" ] [ "boot" "tmp" "tmpfsSize" ])
   ];
 
   options = {
     boot.tmp = {
-      cleanOnBoot = mkOption {
-        type = types.bool;
+      cleanOnBoot = lib.mkOption {
+        type = lib.types.bool;
         default = false;
         description = ''
           Whether to delete all files in {file}`/tmp` during boot.
         '';
       };
 
-      tmpfsSize = mkOption {
-        type = types.oneOf [ types.str types.types.ints.positive ];
+      tmpfsSize = lib.mkOption {
+        type = lib.types.oneOf [ lib.types.str lib.types.types.ints.positive ];
```

Comment:
What is `lib.types.types.ints.positive`?!

Reply:
changed to `lib.types.ints.positive`